### PR TITLE
Fix origin URL for bandcamp embed

### LINKED ISSUE #3252: Incorrect originUrl when scrobbling on the Bandcamp Daily pages
**Describe the bug**
Similarly to #3246, #3249 and #3251, on some Bandcamp pages the scrobbles' `originUrl` is set as the current page URL instead of that of the album being played.
This issue in particular concerns pages under the daily.bandcamp.com subdomain from what I can tell. Some example pages:
- [scene report](https://daily.bandcamp.com/scene-report/klezmer-jewish-folk-feature)
- [features](https://daily.bandcamp.com/features/the-staples-jr-singers-when-do-we-get-paid-interview)
- [tape label report](https://daily.bandcamp.com/tape-label-report/the-tape-label-report-april-2022)
- [album of the day](https://daily.bandcamp.com/album-of-the-day/congotronics-international-wheres-the-one-review)

**How to reproduce**
1. Go to any of the links above
2. Click on an item to play a track
3. See the originUrl in the scrobble

**Expected behavior**
The `originUrl` should be a URL to the album being played rather than the current page the user is on

**Debug logs**
See the `originUrl` (should be `https://congotronicsinternational.bandcamp.com/album/wheres-the-one`):
```
Web Scrobbler: {
  "track": "Where's The One?",
  "artist": "Congotronics International",
  "album": "Where's The One?",
  "albumArtist": null,
  "uniqueID": null,
  "duration": 307,
  "currentTime": null,
  "isPlaying": true,
  "trackArt": "https://f4.bcbits.com/img/a3618193762_16.jpg",
  "isPodcast": false,
  "originUrl": "https://daily.bandcamp.com/album-of-the-day/congotronics-international-wheres-the-one-review"
}
```

**Environment (please complete the following information)**
-   OS: Mac OS
-   Browser: Brave
-   Extension version:  v2.62.0

**Additional context**
On any of these pages, the following DOM selector can target the correct element being played:
`document.querySelector('.playing a').href`

### PR DESCRIPTION
**Pull request by inverse**
Resolves #3252

### CODE REVIEW

**Comment by inverse** [comment]
@MonkeyDo would you be able to review this one? 

I forgot we had an additional connector for the daily pages.

**Review by MonkeyDo — APPROVED** [review]
Works great, thanks for the fix !

File: src/core/content/util.js [hunk 1]
```diff
@@ -745,6 +745,21 @@ const Util = {
 
 		return { artist: null, track };
 	},
+
+	/**
+	 * Get the origin URL from selector, falling back to the document location on failure.
+	 * @param {String} selector
+	 * @returns {String}
```
Comment (MonkeyDo) [inline src/core/content/util.js:752]:
```suggestion
	 * @return {String}
```

File: src/core/content/util.js [hunk 2]
```diff
@@ -745,6 +745,21 @@ const Util = {
 
 		return { artist: null, track };
 	},
+
+	/**
+	 * Get the origin URL from selector, falling back to the document location on failure.
+	 * @param {String} selector
```
Comment (MonkeyDo) [inline src/core/content/util.js:751]:
```suggestion
	 * @param {String} selector - A string containing one or more selectors to match. Must be a valid CSS selector string.
```

**Review by MonkeyDo — COMMENTED** [review]
Placating the JSDocs deities…

**Review by MonkeyDo — APPROVED** [review]
🚀

**Pull request merged**